gdal/doc/source/community/service_providers.rst

```diff
@@ -132,6 +133,12 @@ The steps to add a company to the service providers list are outlined in the :re
    :height: 200 px
    :target: `Spatialys`_
 
+.. _`Faunalia`: https://www.faunalia.eu/
```

Comment:
should be moved under the "Other service providers - use max(width=100px,height=100px) for logo" comment at end of the doc